hw/top_earlgrey/BUILD

```diff
@@ -431,7 +431,7 @@ silicon(
     }),
     rsa_key = select({
         "//signing:owner_key_ecdsa": CLEAR_KEY_SET,
-        "//signing:owner_key_rsa": {"//sw/device/silicon_creator/rom_ext/prodc/keys:keyset": "earlgrey_z0_sival_1"},
+        "//signing:owner_key_rsa": {"//sw/device/silicon_creator/rom_ext/prodc/keys:keyset": "earlgrey_z1_prodc_1"},
```

Comment:
Would you mind correcting this same typo down in the next configuration (`silicon_owner_proda_rom_ext`)?

Reply:
Done